metaflow/runtime.py

```diff
@@ -1108,56 +1108,24 @@ def mark_resume_done(self):
 
     def _get_task_id(self, task_id):
         already_existed = True
+        tags = []
```

Comment:
We need to check these changes with whatever @darinyu is also doing in the #1884 . I like the reduction in lines of code though :)